client/webserver/site/src/js/markets.js

```diff
@@ -959,6 +959,16 @@ export default class MarketsPage extends BasePage {
   handleBookRoute (note) {
```

Comment:
Doing this in `handleBookRoute` will set the price when the book is originally received, but will never update the value. I'm guessing we want the value to update when the book changes.